install.sh

```diff
@@ -279,60 +271,58 @@ for i in $(seq 0 $((${#urls[@]} - 1))); do
   tarfile=$(basename "${url}")
   name="${tarfile%%-*}"   # extract string before first '-'
   rest="${tarfile#*-}"    # extract string after first '-'
-  version="${rest%%-chromeos*}"
-                          # extract string between first '-' and "-chromeos"
+  version="${rest%%-chromeos*}" # extract string between first '-' and "-chromeos"
 
   download_check "${name}" "${url}" "${tarfile}" "${sha256}"
   extract_install "${name}" "${tarfile}"
   update_device_json "${name}" "${version}"
 done
 
 echo_info "\nCreating symlink to 'crew' in ${CREW_PREFIX}/bin/"
-echo -e "${GRAY}"
+
+# Work around https://github.com/chromebrew/chromebrew/issues/3305.
+sudo ldconfig &> /dev/null || true
```

Comment:
Not a fan of `&> /dev/null`.

Reply:
What do you reccomend instead? `&> /dev/null` seems to be doing the job here well enough.

Reply:
I recommend showing the output in case there is an error no matter how ugly it looks.

Reply:
The output is pretty ugly. Is there a way we can show errors but not regular output?

Reply:
Maybe create a log file at `/tmp` and redirect all output to there?

Reply:
Fixed.